src/routes/our-music/index.svelte

```diff
@@ -47,3 +100,142 @@
             allowfullscreen />
     </div>
 {/each}
+
+<center>
+    <h2>
+        <b>Our Playlists</b>
+    </h2>
+</center>
+-->
+
+<br />
+
+<center>
+    <h3>CAC+US Performances</h3>
+</center>
+
+<div class="iframe-container iframe-container-16x9">
+    <iframe
+        title="CAC+US Youtube Playlist"
+        width="560"
+        height="315"
+        src="https://youtube.com/embed/playlist?list=PLac9vFnrjsKMgSrV3XdcHyMKeyX17BP8M"
+        frameborder="0"
+        allow="accelerometer; autoplay; clipboard-write; encrypted-media;
+        gyroscope; picture-in-picture"
+        allowfullscreen />
+</div>
+
+<center>
+    <h3>Spotify Playlist</h3>
```

Comment:
Not exactly Spotify Playlist, can we named it Other Playlist Instead?

Reply:
Done.